web/modules/custom/joinup_migrate/joinup_migrate.install

```diff
@@ -260,3 +260,10 @@ function joinup_migrate_source_db_schema($table) {
 
   return $schema[$table];
 }
+
+/**
+ * Disable the Update module.
+ */
+function joinup_migrate_update_8100() {
```

Comment:
Is there a reason why this not in a `post_update` function? We are just using the API in a straight way, we're not repairing any structure, storage and also we are not upgrading any API that needs backed repairing.